### PR TITLE
docs: fixed spelling mistakes and indentation issue

### PR DESCRIPTION
**Pull request by vvdwivedi**
This fixes the comments mentioned in PR #2093

### CODE REVIEW

File: docs/site/Deploying-with-pm2-and-nginx.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ section of documentation for detailed instructions.
    };
    ```
 
-3. Add pm2 as a dependency to your projet using following command
+3. Add `pm2` as a dependency to your project using following command
```
Comment (b-admike) [inline docs/site/Deploying-with-pm2-and-nginx.md]: nitpick: `using following` -> `using the following`

File: docs/site/Deploying-with-pm2-and-nginx.md [hunk 2]
```diff
@@ -98,16 +98,15 @@ section of documentation for detailed instructions.
 
 ## Deployment
 
-1. Register and start your application with pm2 using following command at the
+1. Register and start your application with `pm2` using following command at the
```
Comment (b-admike) [inline docs/site/Deploying-with-pm2-and-nginx.md]: nitpick: `using following` -> `using the following`

File: docs/site/Deploying-with-pm2-and-nginx.md [hunk 3]
```diff
@@ -128,7 +127,7 @@ section of documentation for detailed instructions.
 
 3. All set! Now you can hit your localhost at `http://localhost:3000/fooapi`
    (assuming nginx is listening to port 80) and your requests will be passed on
-   to pm2 process running your loopback application.
+   to `pm2` process running your loopback application.
```
Comment (b-admike) [inline docs/site/Deploying-with-pm2-and-nginx.md]: nitpick: `loopback` -> `LoopBack`

File: docs/site/Deploying-with-pm2-and-nginx.md (hunk 1 above)
Comment (vvdwivedi) [reply docs/site/Deploying-with-pm2-and-nginx.md]: Done.

File: docs/site/Deploying-with-pm2-and-nginx.md (hunk 3 above)
Comment (vvdwivedi) [reply docs/site/Deploying-with-pm2-and-nginx.md]: Done

File: docs/site/Deploying-with-pm2-and-nginx.md (hunk 2 above)
Comment (vvdwivedi) [reply docs/site/Deploying-with-pm2-and-nginx.md]: Done

**Comment by dhmlau** [comment]
I think the CI failure is not related to this change. Force merging it.
@vvdwivedi , Thanks for your contribution! Your PR has been merged. 🎉

**Pull request merged**